### DISCUSSION TITLE
log not work on appside

### DISCUSSION THREAD

**Discussion by mprediger** [zepp-health/zeppos-forum#40] [Q&A] [answered]
I'm having problems using log on the app-side. It just doesn't show anything that's on the app-side. It works only on the page. Console.log . doesn't work either. Do you know why?

**Comment by MyoungXUE** [comment, marked as answer]
Hello, thanks for your support to the developer forum!

For more information about log debugging please also refer to our documentation at https://docs.zepp.com/docs/guides/best-practice/debug/

The whole process is available, and the logs can be viewed in the corresponding window console on the simulator.

But at the moment you can't see the settings-app on the real machine.

If you still have problems, please describe them in more detail.

**Comment by wimva** [comment]
I'm having this too. I can't get logs working on the app-side. A simple `console.log('test')` doesn't give me a log in the console.

**Reply by MyoungXUE** [reply]
> I'm having this too. I can't get logs working on the app-side. A simple `console.log('test)` doesn't give me a log in the console.

Hello, Can you provide more information to help us reproduce the problem? After our testing, we can see the logs in the Side Service console.

**Reply by wimva** [reply]
I might've not found the 'Side Service Console'? Is this another place than the logs from the device pages?

**Reply by MyoungXUE** [reply]
I guess you may not have found the location of the Side Service Log, please also refer to this document https://docs.zepp.com/docs/guides/best-practice/debug/

It mentions Side Service Log, hope to solve your doubts 😄

**Reply by wimva** [reply]
Oh great 🙈 Thanks for pointing me in the right direction. Works for me there 😄
